bedevere/stage.py

```diff
@@ -100,6 +100,18 @@ async def stage(gh, issue, blocked_on):
     await gh.post(issue["labels_url"], data=[label_name])
 
 
+async def stage_for_review(gh, pull_request):
+    """Apply awaiting review labels."""
+    issue = await util.issue_for_PR(gh, pull_request)
+    username = util.user_login(pull_request)
+    blocked_on = (
+        Blocker.core_review
+        if await util.is_core_dev(gh, username)
+        else Blocker.review
+    )
```

Comment:
```suggestion
    if await util.is_core_dev(gh, username):
        blocked_on = Blocker.core_review
    else:
        blocked_on = Blocker.review
```
I think a regular `if/else` would read better here.